src/test/java/ru/netology/domain/RadioTest.java

```diff
@@ -11,80 +11,77 @@
 
     @Test
     public void validateChangeFields() {
-        assertEquals(0, radio.getCurrentRadioStation());
-        radio.setCurrentRadioStation(4);
-        assertEquals(4, radio.getCurrentRadioStation());
+        Radio radio = new Radio(10, 0, 5, 100, 0, 5);
+        assertEquals(5, radio.getCurrentRadioStation());
     }
 
     @Test
     public void changeOverLastRadioStation() {
-        radio.setCurrentRadioStation(9);
+        Radio radio = new Radio(10, 0, 10, 100, 0, 5);
         radio.pressNextStation();
         assertEquals(0, radio.getCurrentRadioStation());
     }
 
     @Test
     public void changeUnderInitialRadioStation() {
-        radio.setCurrentRadioStation(0);
+        Radio radio = new Radio(10, 0, 0, 100, 0, 5);
         radio.pressPrevStation();
-        assertEquals(9, radio.getCurrentRadioStation());
-
+        assertEquals(10, radio.getCurrentRadioStation());
     }
 
     @Test
     public void nextRadioStation() {
-        radio.setCurrentRadioStation(6);
+        Radio radio = new Radio(10, 0, 5, 100, 0, 5);
         radio.pressNextStation();
-        assertEquals(7, radio.getCurrentRadioStation());
+        assertEquals(6, radio.getCurrentRadioStation());
     }
 
     @Test
     public void prevRadioStation() {
-        radio.setCurrentRadioStation(6);
+        Radio radio = new Radio(10, 0, 5, 100, 0, 5);
         radio.pressPrevStation();
-        assertEquals(5, radio.getCurrentRadioStation());
+        assertEquals(4, radio.getCurrentRadioStation());
     }
 
     @Test
     public void OverInitialRadioStation() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.